gitreceive/receiver/flynn-receive.go

```diff
@@ -96,6 +86,36 @@ Options:
 		return fmt.Errorf("Error getting current app release: %s", err)
 	}
 
+	slugbuilderImageID := os.Getenv("SLUGBUILDER_18_IMAGE_ID")
+	slugrunnerImageID := os.Getenv("SLUGRUNNER_18_IMAGE_ID")
+	stackName := "heroku-18"
+	cfStackName := "cflinuxfs3"
+
+	if stack := prevRelease.Env["FLYNN_STACK"]; stack != "" {
+		switch stack {
+		case "heroku-18":
+		case "cedar-14":
+			fmt.Printf("WARNING: The cedar-14 stack is deprecated and does not receive security updates.")
+			fmt.Printf("WARNING: Unset FLYNN_STACK to use the default stack.")
+			slugbuilderImageID = os.Getenv("SLUGBUILDER_14_IMAGE_ID")
+			slugrunnerImageID = os.Getenv("SLUGRUNNER_14_IMAGE_ID")
+			stackName = "cedar-14"
+			cfStackName = "cflinuxfs3"
```

Comment:
Should this be the legacy version `cflinuxfs2`?

Reply:
Fixed, thanks.